scripts/soroban_repo_to_horizon_repo.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# soroban_repo_to_horizon_repo.sh - given a soroban branch to horizon branch
+#
+# Syntax:   soroban_repo_to_horizon_repo.sh <soroban_branch_name>
+#
+# Examples: soroban_repo_to_horizon_repo.sh main
+
+set -e
+
+if [ "$#" -ne 1 ]; then
+    echo "Syntax: soroban_repo_to_horizon_repo.sh <soroban_branch_name>"
+    exit 1
+fi
+
+SOROBAN_BRANCH=$1
+SOROBAN_REPO_GOMOD=https://raw.githubusercontent.com/stellar/soroban-tools/${SOROBAN_BRANCH}/go.mod
+curl -s ${SOROBAN_REPO_GOMOD} -o soroban.go.mod
+cat soroban.go.mod | grep "github.com/stellar/go " | cut -d \- -f 3
+rm -f soroban.go.mod
```

Comment:
Bit of a nitpick, but you can just pipe to stdout, right? e.g. 

```bash
curl -s ${SOROBAN_REPO_GOMOD} -o - | grep "github.com/stellar/go " | cut -d- -f3
```

This has the benefit of working around any write permission issues.

Reply:
@Shaptic you're not wrong, but it would also require adding `set -o pipefail` upfront, since we don't want an error from the curl to be silenced by the grep.

Reply:
ooh good point :+1: